MekHQ/.editorconfig

```diff
@@ -0,0 +1,9 @@
+# EditorConfig: https://EditorConfig.org
+
+[*]
+end_of_line = crlf
```

Comment:
Three things:
1. I don't know that the project uses `crlf`. I know at least a few us use non-Windows. git should be controlling this for us anyways with core.crlf auto mode. I'd wager these files are actually `lf`.
2. Add `charset = utf-8`
3. Add `insert_final_newline = true`